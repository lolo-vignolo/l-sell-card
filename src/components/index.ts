export { ProductButtons } from '../components/ProductButtons';
export { ProductImage } from '../components/ProductImage';
export { ProductTitle } from '../components/ProductTitle';
export { ProductCard } from '../components/ProductCard';
