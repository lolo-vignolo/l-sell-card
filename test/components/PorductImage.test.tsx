import React from 'react';
import render from 'react-test-renderer';
import { ProductCard } from '../../src/components/ProductCard';
import { ProductImage } from '../../src/components/ProductImage';
import { product_1 } from '../data/product';

describe('PoductTitle', () => {
  test('show the component correctly witht img', () => {
    const wrapper = render.create(<ProductImage img="./coffee-mug.png" />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  test('show the component correctly with default img', () => {
    const wrapper = render.create(
      <ProductCard product={product_1}>
        {() => (
          <>
            <ProductImage />
          </>
        )}
      </ProductCard>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
