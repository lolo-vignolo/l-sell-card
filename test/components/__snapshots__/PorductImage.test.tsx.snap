// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PoductTitle show the component correctly with default img 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product img"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`PoductTitle show the component correctly with default title 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product img"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`PoductTitle show the component correctly witht img 1`] = `
<img
  alt="Product img"
  className="productImg undefined"
  src="./coffee-mug.png"
/>
`;

exports[`PoductTitle show the component correctly witht presonalized title 1`] = `
<img
  alt="Product img"
  className="productImg undefined"
  src="./coffee-mug.png"
/>
`;
